scripts/linux-aarch64/install.sh

```diff
@@ -11,12 +13,75 @@ read -p "Absolute install directory path (default: $HOME/albyhub): " USER_INSTAL
 
 INSTALL_DIR="${USER_INSTALL_DIR:-$HOME/albyhub}"
 
+verify_package() {
```

Comment:
can this function be moved to a separate script file?

Reply:
then another file needs to be downloaded (and actually verified 🤔 ) 

I'd keep those simple